sprites.py

```diff
@@ -279,7 +279,7 @@ def set_layer(self, layer=None):
     def set_label(self, new_label, i=0):
         ''' Set the label drawn on the sprite '''
         self._extend_labels_array(i)
-        if isinstance(new_label, str) or isinstance(new_label, unicode):
+        if isinstance(new_label, str) or isinstance(new_label, str):
```

Comment:
Second half of the conditional expression is the same as the first.

Reply:
Fixed in 7dca42e.